compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParser.java

```diff
@@ -1978,11 +1989,10 @@ private STNode parseParameter(SyntaxKind prevParamKind, boolean isParamNameOptio
         }
 
         STNode type = parseTypeDescriptor(ParserRuleContext.TYPE_DESC_BEFORE_IDENTIFIER);
```

Comment:
Parsing the inclusionSymbol in a single place like this is cleaner to me
```suggestion
        STNode inclusionSymbol = parseInclusionSymbol();
        STNode type = parseTypeDescriptor(ParserRuleContext.TYPE_DESC_BEFORE_IDENTIFIER);
```